apps/full-site-editing/full-site-editing-plugin/block-inserter-modifications/tips/index.scss

```diff
@@ -0,0 +1,3 @@
+.block-editor-inserter__panel-content .components-tip {
+	margin-top: 20px;
```

Comment:
How come the default styling in the editor didn't work?

Reply:
> How come the default styling in the editor didn't work?

There isn't a default style for contextual tips. They don't exist in core, yet :-D

> I'd try to avoid these editor overrides

It doesn't override any styles. Maybe we can add a custom CSS class.